desc/compute/utils.py

```diff
@@ -1212,6 +1212,112 @@ def surface_integrals_transform(grid, surface_label="rho"):
     return surface_integrals_map(grid, surface_label, expand_out=False)
 
 
+def surface_variance(
+    grid,
+    q,
+    weights=jnp.array([1.0]),
+    bias=False,
+    surface_label="rho",
+    expand_out=True,
+):
+    r"""Compute the weighted sample variance of ``q`` on each surface of the grid.
+
+    Computes the following quantity on each surface of the grid.
+
+    .. math::
```

Comment:
minor point, but I've been trying to use unicode instead of TeX for math notation in docstrings, so that its readable both in source and formatted form. There's a handy website for converting stuff: https://www.unicodeit.net/